controller/control.go

```diff
@@ -1035,3 +1038,16 @@ func (c *Controller) monitoring(address string, backend types.Backend) {
 	logrus.Infof("Monitoring stopped %v", address)
 	c.RemoveReplicaNoLock(address)
 }
+
+func (c *Controller) monitorPreload(address string) {
+	c.Lock()
```

Comment:
Start controller and 3 replicas, immediately restart all 3 replicas.
Between the above 2 events, controller sends start signal to replica 1, and it is waiting for replica 1 to complete its preload with the lock and sleeping for 2 sec.
Replicas got restarted during this 2 sec, and the old IPs persist.
Controller again tries to get preload status on same IP after 2 sec, since controller doesn't know that the replica went down while it was sleeping. 
Preload status received by controller "Preload is not done yet on backend tcp://172.18.0.5:9502, current status: NA" Error at replica for the the above preloadStatus request "Fail to get revision counter: BUG: revision file wasn't initialized".
Controller tries it indefinitely while the GET request from replicas to controller are waiting for this lock.